api_tests/webgui/webgui.ts

```diff
@@ -70,6 +70,18 @@ setTimeout(async function() {
             );
         });
 
+        it("[David] Sets appropriate CORS headers on error responses", async () => {
+            let res = await request(david.comit_node_url())
+                .get("/swaps/rfc003/deadbeef-dead-beef-dead-deadbeefdead")
+                .set("Origin", "http://localhost:8080");
+
+            res.should.have.status(404);
```

Comment:
🤡  @thomaseizinger would comment "please use expect here".

Reply:
Fair :D

Reply:
I rewrote all assertions in `webgui` to use `expect`.